src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -9,53 +9,117 @@
 import javafx.scene.input.ClipboardContent;
 import javafx.stage.Stage;
 import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.AddBookingCommand;
+import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.ClearCommand;
+import seedu.address.logic.commands.DeleteBookingCommand;
+import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.EditBookingCommand;
+import seedu.address.logic.commands.EditCommand;
+import seedu.address.logic.commands.ExitCommand;
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.StatusCommand;
+import seedu.address.logic.parser.CliSyntax;
+import seedu.address.logic.parser.Prefix;
+
 
 /**
  * Controller for a help page
  */
 public class HelpWindow extends UiPart<Stage> {
+    public static final Prefix NAME = CliSyntax.getPrefixName();
+    public static final Prefix PHONE = CliSyntax.getPrefixPhone();
+    public static final Prefix BOOKING = CliSyntax.getPrefixBooking();
+    public static final Prefix RA = CliSyntax.getPrefixResidenceAddress();
+    public static final Prefix CLEAN = CliSyntax.getPrefixCleanStatusTag();
+    public static final Prefix TAG = CliSyntax.getPrefixTag();
+    public static final Prefix START = CliSyntax.getPrefixBookingStartDate();
+    public static final Prefix END = CliSyntax.getPrefixBookingEndDate();
+    public static final Prefix RESIDENCE = CliSyntax.getPrefixResidence();
+
 
     public static final String USERGUIDE_URL = "https://github.com/AY2021S2-CS2103-T16-3/"
                                              + "tp/blob/master/docs/UserGuide.md";
 
-    public static final String ADD_PROMPT = "add - Adds a residence into ResidenceTracker."
-                                          + "\nEnter \"add n/NAME_OF_RESIDENCE a/ADDRESS "
-                                          + "[clean/[y or n]] [t/TAG]...\""
-                                          + "\nExample: add n/Clementi HDB a/459A Clementi Ave 3, #04-257, S121459 "
-                                          + "clean/n t/Reserved\n\n";
+    public static final String ADD_PROMPT = AddCommand.getCommandWord()
+                                          + " - Adds a residence into ResidenceTracker."
+                                          + "\nEnter \"" + AddCommand.getCommandWord()
+                                          + " " + NAME + "NAME_OF_RESIDENCE "
+                                          + RA + "ADDRESS "
+                                          + "[" + CLEAN + "[y or n]]"
```

Comment:
I'm thinking maybe instead of showing c/[y or n] we can write c/VALID_CLEAN_STATUS and then mention that VALID_CLEAN_STATUS for clean: 'y' or 'clean' and unclean: 'n' or 'unclean'?